llama_index/storage/kvstore/simple_kvstore.py

```diff
@@ -18,13 +19,18 @@ class SimpleKVStore(BaseInMemoryKVStore):
     """Simple in-memory Key-Value store.
 
     Args:
-        persist_path (str): path to persist the store
-
+        data (Optional[DATA_TYPE]): data to initialize the store with
+        fs (Optional[fsspec.AbstractFileSystem]): filesystem to use. defaults to local storage
     """
 
-    def __init__(self, data: Optional[DATA_TYPE] = None) -> None:
+    def __init__(
+        self,
+        data: Optional[DATA_TYPE] = None,
+        fs: Optional[fsspec.AbstractFileSystem] = None,
```

Comment:
since `fs` is passed when calling `persist`. This is not necessary right?

Reply:
Hm, I agree with you, it's not necessary. In my mind it is better for the storage context to remain consistent -- if you instantiate the object from a certain persistdir, we should default to the persistdir being the same. And if the persist dir is consistent, then the fs also needs to be consistent.

But I guess that isn't how the API is designed now anyways -- the persist_path is not stored.